$OpenBSD: patch-scribus_ui_colorlistmodel_cpp,v 1.2 2021/04/07 14:08:35 sthen Exp $

avoid segfault caused by CommonStrings::None by removing m_NoneColor, which is
unused anyways

Index: scribus/ui/colorlistmodel.cpp
--- scribus/ui/colorlistmodel.cpp.orig
+++ scribus/ui/colorlistmodel.cpp
@@ -31,8 +31,6 @@ ColorPixmapValue::ColorPixmapValue( const ScColor& col
 	m_name = colName;
 }
 
-ColorPixmapValue ColorListModel::m_NoneColor(ScColor(), nullptr, CommonStrings::None);
-
 ColorListModel::ColorListModel(QObject *parent)
 	          : QAbstractItemModel(parent)
 {
